src/main/scala/edu/cmu/ml/rtw/pra/features/PraFeatureGenerator.scala

```diff
@@ -1,19 +1,16 @@
 package edu.cmu.ml.rtw.pra.features
 
-import edu.cmu.ml.rtw.pra.data.Dataset
-import edu.cmu.ml.rtw.pra.data.NodePairInstance
+import edu.cmu.ml.rtw.pra.data.{Dataset, NodePairInstance, Instance}
```

Comment:
This added import is unused - I think you can just revert all of the changes to this file.